packages/perception/deps/33-openvino.deps

```diff
@@ -26,12 +26,18 @@ build_openvino()
   cd l_openvino_toolkit_p_2019.3.334
   sudo  ./install_openvino_dependencies.sh
   sed -i 's/ACCEPT_EULA=decline/ACCEPT_EULA=accept/g' silent.cfg
-  if [ ! -d /opt/intel/openvino ];then
+  openvino_version=$(ls -l /opt/intel/openvino |awk -F '\/' '{print $7}')
```

Comment:
Could we use "readlink" command, to check if softlink exist and is match target version.
readlink /opt/intel/openvino